scripts/typescript_checks.py

```diff
@@ -57,13 +811,123 @@ def validate_compiled_js_dir() -> None:
             'in %s: %s' % (COMPILED_JS_DIR, TSCONFIG_FILEPATH, out_dir))
 
 
+@contextlib.contextmanager
+def compile_strict_tsconfig(
+        config_path: str,
+        error_messages: List[str]
```

Comment:
If you're going to un-indent the last line of the function signature, then these lines only need to be indented by 4 spaces

Reply:
Done